cabal.project

```diff
@@ -0,0 +1,3 @@
+packages: .
```

Comment:
Is this required for this to build correctly or was this just done to accelerate the build?

Reply:
i added the cabal.project file to pass flags to semirings, because i hate doing it in the .cabal file (it looks uglier). 

Unless you're referring to the specific line there, and not the addition of the file, in which case, i believe that in the presence of a cabal.project file you will get a complaint without including that line, because cabal will complain about missing targets. Example from `semirings`, where i removed the line:

```bash
$ cabal new-build
cabal: No targets given and there is no package in the current directory. Use
the target 'all' for all packages in the project or specify packages or
components by name or location. See 'cabal build --help' for more details on
target options.
```

Reply:
I was referring to the whole file, not just the line. My understanding of what you said is that it is just done to accelerate the builds locally. Putting it in the cabal file itself would have a very different (and very bad) effect. Go ahead and leave this as is for now, but I will probably move the flags for semirings into a nonstandardly-named cabal project file later and have the build script refer to it.

Reply:
oh sorry, i didn't actually answer your question. the answer is yes, to accelerate the builds.

Reply:
marking as resolved